selfdrive/car/toyota/interface.py

```diff
@@ -140,6 +140,16 @@ def get_params(candidate, fingerprint):
       ret.mass = 4607 * CV.LB_TO_KG + std_cargo #mean between normal and hybrid limited
       ret.steerKpV, ret.steerKiV = [[0.6], [0.05]]
       ret.steerKf = 0.00006
+      
+    elif candidate == CAR.SIENNA:
+      ret.safetyParam = 100
+      ret.wheelbase = 3.03 # 119.3in =  3030.22mm
+      ret.steerRatio = 16.0
+      tire_stiffness_factor = 0.444 # not optimized yet
+      ret.mass = 4590 * CV.LB_TO_KG + std_cargo #mean between normal (4505) and limited (4750)
+      ret.steerKpV, ret.steerKiV = [[0.18], [0.0075]]
+      ret.steerKf = 0.0003
```

Comment:
this tuning seems pretty weird :). You sure that it's better than, for example, the one from the Highlander?

Reply:
it's not as jerky with this tune but still needs corner tuning

Reply:
@vanillagorillaa what values errors?

Reply:
Added to readme. working on values now